samples/dapr/AppHost/aspire-manifest.json

```diff
@@ -1,8 +1,12 @@
 {
   "resources": {
+    "statestore": {
+      "type": "dapr.component.v0",
+      "componentType": "statestore"
+    },
     "servicea": {
       "type": "project.v0",
-      "path": "..\\ServiceA\\DaprServiceA.csproj",
+      "path": "../ServiceA/DaprServiceA.csproj",
```

Comment:
@mitchdenny we should fix this and normalize to forward slashes. Hopefully it doesn't break @ellismg ?

Reply:
https://github.com/dotnet/aspire/issues/899